<launch>
    <node pkg="movel_laser_filters" type="scan_to_scan_filter_chain" output="screen" name="scan_filter_chain">
        <rosparam command="load" file="$(find movel_laser_filters)/examples/profiles.yaml" />
    </node>
</launch>